app/Components/AlertBox.js

```diff
@@ -0,0 +1,86 @@
+import { View, Modal, StyleSheet, Text, Button, Pressable } from "react-native";
+
+const AlertBox = ({
+  alertVisible,
+  setAlertVisible,
+  deleteVector,
+  targetId,
+}) => {
+  const handleDelete = () => {
+    setAlertVisible(false);
+    deleteVector(targetId);
+  };
+
+  return (
+    <Modal
+      style={styles.modal}
+      visible={alertVisible}
+      transparent={true}
+      animationType="fade"
+    >
+      <Pressable style={styles.backdrop}/>
+      <View style={styles.box}>
+        <Text style={{ fontSize: 20, textAlign: "center" }}>
+          Do you want to delete this circle?
+        </Text>
+        <View style={styles.buttonContainer}>
+          <Button
+            color="#2A6620"
+            title="DELETE"
+            onPress={() => handleDelete()}
+          />
+          <Button
+            color="#2A6620"
+            title="CANCLE"
```

Comment:
Minor spelling correction for `cancel`

Reply:
Thanks you for pointing that out! Just fixed it!